contrib/build-push-ceph-container-imgs.sh

```diff
@@ -122,26 +141,37 @@ function create_head_or_point_release {
 
     # find branch associated to that tag
     BRANCH=$(git branch -r --contains tags/"${tag_to_build[*]}" | grep -Eo 'stable-[0-9].[0-9]')
-    echo "Building a release Ceph container image based on branch $BRANCH and tag ${tag_to_build[*]}"
-    RELEASE="${tag_to_build[*]}-$BRANCH"
+    echo "Building a release Ceph container image based on branch $CONTAINER_BRANCH and tag ${tag_to_build[*]}"
+    RELEASE="${tag_to_build[*]}-$CONTAINER_BRANCH"
     # (todo): remove this when we have a better solution like running
     # the build script directly from the right branch.
-    if [ "${BRANCH}" == "stable-3.2" ]; then
+    if [ "${CONTAINER_BRANCH}" == "stable-3.2" ]; then
       CEPH_RELEASES=(luminous mimic)
-    elif [ "${BRANCH}" == "stable-4.0" ]; then
+    elif [ "${CONTAINER_BRANCH}" == "stable-4.0" ]; then
       CEPH_RELEASES=(nautilus)
     fi
   else
     set -e
-    echo "Building a devel Ceph container image based on branch $BRANCH and commit $LATEST_COMMIT_SHA"
-    RELEASE="$BRANCH-$LATEST_COMMIT_SHA"
+    echo "Building a devel Ceph container image based on branch $CONTAINER_BRANCH and commit $CONTAINER_SHA"
+    RELEASE="$CONTAINER_BRANCH-$CONTAINER_SHA"
   fi
 }
 
 declare -F build_ceph_imgs  ||
 function build_ceph_imgs {
   echo "Build Ceph container image(s)"
-  make CEPH_DEVEL=${DEVEL} RELEASE="$RELEASE" build.parallel
+  FLAVORS=""
+  if ${CI_CONTAINER}; then
+    FLAVORS="${CEPH_BRANCH},centos,7"
+    make FLAVORS=${FLAVORS} \
+         CEPH_DEVEL="true" \
+         RELEASE=${RELEASE} \
+         TAG_REGISTRY=${CONTAINER_REPO_ORGANIZATION} \
+         IMAGES_TO_BUILD=daemon-base \
+         build.parallel
+  else
+    make CEPH_DEVEL=${DEVEL} RELEASE=${RELEASE} TAG_REGISTRY=${CONTAINER_REPO_ORGANIZATION} build.parallel
```

Comment:
I don't see `CONTAINER_REPO_ORGANIZATION` declare anywhere if `CI_CONTAINER` is false, so I think that going to break. Can we just use `make CEPH_DEVEL=${DEVEL} RELEASE="$RELEASE" build.parallel` since we are in the `else`? Thanks